packages/patternfly-4/react-table/src/components/Table/SelectColumn.tsx

```diff
@@ -4,13 +4,13 @@ export interface SelectColumnProps {
   name?: string;
   children?: React.ReactNode;
   className?: string;
-  onSelect?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onSelect?: (event: React.FormEvent<HTMLInputElement>) => void;
 }
 
 export const SelectColumn: React.FunctionComponent<SelectColumnProps> = ({
   children = null as React.ReactNode,
   className = '',
-  onSelect = null as (event: React.ChangeEvent<HTMLInputElement>) => void,
+  onSelect = null as (event: React.FormEvent<HTMLInputElement>) => void,
```

Comment:
This changes our exported types.

Reply:
Yes, this was intentional. Do you happen to know why OnSelect used `React.MouseEvent` while SelectColumn onSelect method was using `React.ChangeEvent<HTMLInputElement>`? It seems like general misalignment but figured it's worth asking about.